### PR TITLE
Preflight 1

### PR DESCRIPTION
**Pull request by wanda-carlson**
> This is a quote with [google](https://www.google.com)

### CODE REVIEW

**Comment by wanda-carlson** [comment]
Screen Shot 2022-01-06 at 7 59 26 AM
Do you see this? @henry-richard

**Comment by wanda-carlson** [comment]
I think you can see it **Henry Richard**

**Comment by henry-richard** [comment]
I am henry

File: src/foo.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # The Foo class v2
 class Foo:
   def hello():
-    print("Hello foo")
```
Comment (henry-richard) [inline src/foo.py:4]: dont remove this

File: src/foo.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
+    print("Hello foo1")
```
Comment (henry-richard) [inline src/foo.py:4]: add a new line

**Review by henry-richard — CHANGES_REQUESTED** [review]
minor changes @wanda-carlson

**Review by henry-richard — APPROVED** [review]
Awesome!

**Pull request merged**